docs/howtoguides/enable_fips.rst

```diff
@@ -54,7 +63,7 @@ If you wish to disable FIPS, you can use the following command:
 
 .. code-block:: bash
 
-    sudo pro disable fips
+    sudo pro disable fips-updates
```

Comment:
Does the difficulty in removing fips also apply to fips-updates?

Reply:
yes